packages/arb-validator/coordinator/server.go

```diff
@@ -57,6 +57,17 @@ func NewServer(
 	connectionInfo ethbridge.ArbAddresses,
 	ethURL string,
 ) *Server {
+	return CreateVM(NewCoordinator(machine, key, validators, connectionInfo, ethURL))
```

Comment:
I'm on board with refactoring this chunks since it was a lot of code doing pretty different things, but do these new functions have to be public?